### PR TITLE
Turn off regression tests for sphere_aquaplanet_rhoe_equilmoist_allsky_gw_raw_zonallyasymmetric

### PR DESCRIPTION
**Pull request by nefrathenrici**
#2704 cause nondeterministic failures, see [here](https://buildkite.com/clima/climaatmos-ci/builds/16914#018de383-5fb2-46c0-a813-4eb4849a51e9).

This PR turns off regression tests for the failing job until we fix

### CODE REVIEW

File: src/parameterized_tendencies/radiation/RRTMGPInterface.jl [hunk 1]
```diff
@@ -1210,30 +1210,12 @@ NVTX.@annotate function update_sw_fluxes!(
...
-            model.face_clear_lw_flux .+ model.face_clear_sw_flux,
-        )
-    model.face_flux .=
-        ifelse.(
-            model.cos_zenith' .<= sqrt(eps(FT)),
-            model.face_lw_flux,
-            model.face_lw_flux .+ model.face_sw_flux,
-        )
+        model.face_clear_lw_flux .+ model.face_clear_sw_flux
+    model.face_flux .= model.face_lw_flux .+ model.face_sw_flux
```
Comment (charleskawczynski) [inline src/parameterized_tendencies/radiation/RRTMGPInterface.jl:1218]: Do we need to revert this part?

File: src/parameterized_tendencies/radiation/RRTMGPInterface.jl [hunk 2]
```diff
@@ -1210,30 +1210,12 @@ NVTX.@annotate function update_sw_fluxes!(
...
-    FT = eltype(model.face_flux)
-    model.face_flux .=
-        ifelse.(
-            model.cos_zenith' .<= sqrt(eps(FT)),
-            model.face_lw_flux,
-            model.face_lw_flux .+ model.face_sw_flux,
-        )
-
-end
+update_net_fluxes!(_, model) =
```
Comment (charleskawczynski) [inline src/parameterized_tendencies/radiation/RRTMGPInterface.jl:1213]: Do we need to revert this part?

**Comment by szy21** [comment]
Could you open an issue to track this so we don't forget?

**Comment by szy21** [comment]
Ah, you need to remove this [line](https://github.com/CliMA/ClimaAtmos.jl/blob/02e8c52f564760accbd39d2811fa2ea2440d5908/.buildkite/pipeline.yml#L301)

**Comment by szy21** [comment]
Hmm, it seems you need to remove the relevant job in the mse table as well.

**Pull request merged**